pkg/apis/core/validation/validation.go

```diff
@@ -2824,6 +2824,19 @@ func validateDNSPolicy(dnsPolicy *core.DNSPolicy, fldPath *field.Path) field.Err
 	return allErrors
 }
 
+func validateFSGroupChangePolicy(fsGroupPolicy *core.PodFSGroupChangePolicy, fldPath *field.Path) field.ErrorList {
+	allErrors := field.ErrorList{}
+	switch *fsGroupPolicy {
+	case core.AlwaysChangeVolumePermission, core.OnRootMismatch:
+	case "":
+		allErrors = append(allErrors, field.Required(fldPath, ""))
```

Comment:
I don't think we should special case this - the field is NOT required, the value "" is invalid

Reply:
fixed